app/controllers/cogy/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Cogy
   class ApplicationController < ActionController::Base
     # @todo https://github.com/skroutz/cogy/issues/43
-    skip_before_action :verify_authenticity_token
+    protect_from_forgery unless: -> { request.format.json? }
```

Comment:
After [I thought I found](https://github.com/rails/rails/issues/29284) an error in the documentation, it turned out there is something that I don't understand about how this works. Anyway... 

Calling `protecte_from_forgery` before skipping the callback does not break the implementation.